broadcast_formats.md

```diff
@@ -67,8 +75,7 @@ of the website of the URL.
 # Data format 3 (also known as RAWv1)
 **Lifecycle: In production.**
 
-RAWv1 is the primary dataformat in 1.x branch firmware. Will be
-depreciated in 2.x branch firmware. RAWv1 has been default format in tags shipped from June 2018 onwards. 
+RAWv1 is a primary data format in 1.x and 2.x branch firmware. It includes a 1 byte humidity field and 2 byte temperature, atmospheric pressure, X/Y/Z accelerometer and battery fields.  RAWv1 has been included in tags shipped since June 2018 . 
```

Comment:
Long term goal is to phase out the format 3 as format 5 is superior in every way. Maybe this could be phrased better than "will be depreciated"?

Reply:
Did you notice my comment regarding omitting the format 5 example in the general discussion.

Reply:
I'm not sure,  do you mean Slack or somewhere else?

Reply:
Embedded in this (broadcast_formats.md) file I put a question asking if 
it is better to omit the example of the format 5 data here in deference to the format specific documentation.

I seem to be having some trouble viewing the completed file, 
so go ahead commit the changes. 
Thanks
PS I am not sure what the [Resolve conversation] box below this is for.

Reply:
The example data is there to demonstrate how to parse the manufacturer specific data payload, let's leave it in place. Resolve conversation button marks the conversation as decided, I think the conversation will be stored but I'm not 100 % certain. This conversation can be open until the PR has been cleared up and merged.